Tests/Packet++Test/main.cpp

```diff
@@ -2105,6 +2105,9 @@ PTF_TEST_CASE(HttpRequestLayerCreationTest)
 	PTF_ASSERT(httpPacket.addLayer(&httpLayer), "Adding HTTP request layer failed");
 	hostField->setFieldValue("www.ynet.co.il");
 	httpLayer.getFirstLine()->setMethod(HttpRequestLayer::HttpGET);
+	PTF_ASSERT(httpLayer.getFirstLine()->getMethod() == HttpRequestLayer::HttpGET, "Couldn't set method");
+	httpLayer.getFirstLine()->setVersion(pcpp::HttpVersion::OneDotOne);
+	PTF_ASSERT(httpLayer.getFirstLine()->getVersion() == pcpp::HttpVersion::OneDotOne, "Couldn't set version");

```

Comment:
these doesn't compile as you can see here:
https://travis-ci.org/seladb/PcapPlusPlus/jobs/603934668#L325

, please `pcpp::HttpVersion::OneDotOne` with `OneDotOne`